src/request.rs

```diff
@@ -108,6 +111,8 @@ impl RequestBuilder {
             body: [],
             max_redirections: 5,
             follow_redirects: true,
+            connect_timeout: Duration::from_secs(30),
```

Comment:
@sbstp I did not pull this out into a constant to stay consistent with e.g. `max_redirections`. If we want to pull out all of them, I think a separate PR would be nicer to keep this as simple as possible.

Reply:
Yeah that's fine.